.github/workflows/build_wheels_windows.yml

```diff
@@ -202,10 +202,10 @@ jobs:
           ${CONDA_RUN} pip install "${{ inputs.repository }}/dist/$WHEEL_NAME"
           if [[ ! -f "${{ inputs.repository }}"/${SMOKE_TEST_SCRIPT} ]]; then
             echo "${{ inputs.repository }}/${SMOKE_TEST_SCRIPT} not found"
-            ${CONDA_RUN} ${ENV_SCRIPT} python -c "import ${PACKAGE_NAME}; print('package version is ', ${PACKAGE_NAME}.__version__)"
+            ${CONDA_RUN} "${{ inputs.repository }}/${ENV_SCRIPT}" python -c "import ${PACKAGE_NAME}; print('package version is ', ${PACKAGE_NAME}.__version__)"
```

Comment:
It might be easier to avoid similar mistakes if you can set `working-directory: ${{ inputs.repository }}` like the previous step

Reply:
Thanks @huydhn, agree with you, but I don't have test env in local, so let's fix it firstly with the smallest change and then we can have another enhance PR to simplify the smoke test part. How do you think?